google/cloud/aiplatform/tensorboard/plugins/tf_profiler/profile_uploader.py

```diff
@@ -38,20 +38,18 @@
 import tensorflow as tf
 
 from google.cloud import storage
-from google.cloud.aiplatform.compat.services import tensorboard_service_client_v1beta1
+from google.cloud.aiplatform.compat.services import tensorboard_service_client_v1
+from google.cloud.aiplatform.compat.types import tensorboard_data_v1 as tensorboard_data
```

Comment:
We append the `*_v1beta1` i.e. `tensorboard_service_client_v1beta1` only when the `v1beta1` version is in use which is against compat's `DEFAULT_VERSION = V1`. 

For all `compat` supporting `types` and `services`, you can go with i.e 
`from google.cloud.aiplatform.compat.types import tensorboard_data` where `types.tensorboard_data = types.tensorboard_data_v1` see config here: https://github.com/googleapis/python-aiplatform/blob/main/google/cloud/aiplatform/compat/__init__.py#L24

The lines below should be sufficient for calling v1 as the default version:
```
from google.cloud.aiplatform.compat.services import tensorboard_service_client
from google.cloud.aiplatform.compat.types import tensorboard_data
```

Reply:
ok changed. Thanks!